.github/workflows/auto-release.yml

```
@@ -112,30 +120,17 @@ jobs:
-
-          # Add changes and commit
+          # Add our changes and commit
           git add package.json CHANGELOG.md
           git commit -m "chore: release v${{ steps.version.outputs.version }} [skip ci]"
 
           # Push with lease protection to avoid overwriting concurrent changes
-          git push --force-with-lease origin main
-
-      - name: Create Git tag
+          git push --force-with-lease origin main      - name: Create Git tag
         run: |
```

actionlint (1.7.7)

key "run" is duplicated in element of "steps" section. previously defined at line:124,col:9

(syntax-check)